ASM/training-mode/Onscreen Display/Act OoJump/Display Act OoJump Frame Count - Static Function.asm

```diff
@@ -0,0 +1,118 @@
+    # To be inserted at 8000550c
+    .include "../../Globals.s"
+    .include "../../../m-ex/Header.s"
+
+    .set playerdata, 31
+    .set player, 30
+
+    backup
```

Comment:
This backup and the following restore are redundant and can be removed. When you remove them, replace with a comment explaining why there is no backup: function is called by other injection which handles register backup.

Reply:
I tried remove backup/restore in this static function, but it seems to occur freeze when act after jump Idk why.

So I think these backup/restore don't need to remove.